fusesoc/core.py

```diff
@@ -3,40 +3,12 @@
 # SPDX-License-Identifier: BSD-2-Clause
 
 import logging
-import warnings
 
-from fusesoc.capi1.core import Core as Capi1Core
 from fusesoc.capi2.core import Core as Capi2Core
 
 logger = logging.getLogger(__name__)
 
 
 class Core:
     def __new__(cls, *args, **kwargs):
-        with open(args[0]) as f:
-            l = f.readline().split()
-            if l:
-                first_line = l[0]
-            else:
-                first_line = ""
-            if first_line == "CAPI=1":
-                warnings.warn(
-                    "The CAPI1 core file description format is deprecated and "
-                    "will be removed in the next major version of FuseSoC. "
-                    "Please port your core files to the CAPI2 format.",
-                    FutureWarning,
-                )
-                return Capi1Core(*args, **kwargs)
-            elif first_line == "CAPI=2:":
-                return Capi2Core(*args, **kwargs)
-            else:
```

Comment:
I think we should keep the logic for deciding capi version by looking at the first line.